### PR TITLE
Cnf 518

### PR DESCRIPTION
**Pull request by StephenJamesSmith**
[CNF-518](https://issues.redhat.com/browse/CNF-518) for v4.6.

### CODE REVIEW

**Comment by MarSik** [comment]
Same here @StephenJamesSmith, why does the PR include the multiple hugepages and the other content? Shouldn't it be just about discovery mode?

File: modules/cnf-running-cnf-tests-in-discovery-mode.adoc [hunk 1]
```diff
@@ -0,0 +1,274 @@
+// CNF-518 Running CNF tests in discovery mode
```
Comment (fedepaol) [inline modules/cnf-running-cnf-tests-in-discovery-mode.adoc:1]: Why not extending the current doc for cnf tests? 
I mean the one under https://docs.openshift.com/container-platform-ocp/4.5/scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes.html#cnf-performing-end-to-end-tests-for-platform-verification_cnf-master 

Here there are infos that are duplicated from there, others are missing.

File: modules/cnf-running-cnf-tests-in-discovery-mode.adoc [hunk 2]
```diff
@@ -0,0 +1,274 @@
...
+// Module included in the following assemblies:
+//
+// *.adoc
+
+[id="cnf-running-cnf-tests-in-discovery-mode_{context}"]
+= Running CNF tests in discovery mode
+
+The Cloud-native Network Functions https://quay.io/repository/openshift-kni/cnf-tests?tag=latest&tab=tags[(CNF)] tests image is a containerized
```
Comment (fedepaol) [inline modules/cnf-running-cnf-tests-in-discovery-mode.adoc:9]: This should reference the downstream image.

File: modules/cnf-associate-secondary-interfaces-metrics-to-network-attachments.adoc [hunk 3]
```diff
@@ -0,0 +1,86 @@
+// CNF-43 Associate Secondary Interfaces Metrics to Network Attachments
```
Comment (fedepaol) [inline modules/cnf-associate-secondary-interfaces-metrics-to-network-attachments.adoc:1]: This is "generic openshift", not related only to CNF
I think the best place is under the multiple network section, which is what these metrics are about https://docs.openshift.com/container-platform/4.5/networking/multiple_networks

File: modules/cnf-running-cnf-tests-in-discovery-mode.adoc [hunk 4]
```diff
@@ -0,0 +1,274 @@
...
+
+[source,yaml]
+----
+apiVersion: performance.openshift.io/v1
+kind: PerformanceProfile
+metadata:
+  name: performance
+spec:
+  cpu:
+    isolated: "0-15"
```
Comment (fedepaol) [inline modules/cnf-running-cnf-tests-in-discovery-mode.adoc:227]: This must be corrected (also upstream) with
    isolated: "4-15"
    reserved: "0-3"

**Comment by ahardin-rh** [comment]
Closing in favor of https://github.com/openshift/openshift-docs/pull/26429 after discussion with Stephen

**Pull request closed without merging**